src/Simulations/time_step_wizard.jl

```diff
@@ -70,7 +70,13 @@ function TimeStepWizard(FT=Float64;
                         min_Δt = 0.0,
                         cell_advection_timescale = cell_advection_timescale,
                         cell_diffusion_timescale = infinite_diffusion_timescale)
-
+    
+    # check if user gave max_change or min_change values that are invalid
+    if min_change >= 1.0 || max_change <= 1.0
```

Comment:
```suggestion
    if min_change >= 1 || max_change <= 1
```

Reply:
I think it's better to make these separate checks. Ie check `min_change` and then afterwards check `max_change`. The error message will be more specific.

Reply:
Yes, that aligns with what I suggested -- already implemented ;)